content/en/blog/2025/go-compile-time-instrumentation.md

```diff
@@ -0,0 +1,92 @@
+---
+title:
+  Alibaba, Datadog and Quesma Join Forces on Go Compile-Time Instrumentation
+linkTitle: Go Compile-Time Instrumentation
+date: 2025-01-17
+author: OpenTelemetry Governance Committee
+issue: https://github.com/open-telemetry/community/issues/2509
+sig: Governance Committee
+cSpell:ignore: instrgen quesma toolexec
+---
+
+Standards are only useful if they're widely adopted, and adoption is only
+effective if the available tooling facilitates it. I imagine
+[SI units](https://en.wikipedia.org/wiki/International_System_of_Units) would
+not have been too popular when they were introduced if you had to build your own
+scales to weigh things in Kilograms!
+
+If you use [OpenTelemetry in Go](/docs/languages/go/), you'll be familiar with
+the challenges of configuring instrumentation libraries to automatically
+generate telemetry from well-known open source components. Due to the compiled
+nature of the language, you currently have two options (unless you want to
+"build your own scales", or the OTel equivalent):
+
+- Use a separate binary that analyzes your Go process and attaches eBPF programs
+  to hooks in your application (see
+  [opentelemetry-go-instrumentation](https://github.com/open-telemetry/opentelemetry-go-instrumentation/)).
+- Manually configure instrumentation libraries in your code (see an
+  [example](/docs/languages/go/getting-started/#instrument-the-http-server) to
+  instrument `net/http`).
```

Comment:
I'd vote for a comma or m-dashe instead of parenthetical remarks. E.g.,

```suggestion
- Use a separate binary that analyzes your Go process and attaches eBPF programs
  to hooks in your application &mdash; see
  [opentelemetry-go-instrumentation](https://github.com/open-telemetry/opentelemetry-go-instrumentation/).
- Manually configure instrumentation libraries in your code, for example see
  [Instrument the HTTP server](/docs/languages/go/getting-started/#instrument-the-http-server).
```